packages/flutter/test/widgets/shadow_test.dart

```diff
@@ -52,15 +53,14 @@ void main() {
         ),
       );
     }
-    for (int elevation in kElevationToShadow.keys) {
```

Comment:
Why is this removed?

Reply:
You caught me in a temporary commit where I wanted to speed up triaging golden file changes.

The test aborted after the first golden failure. As you see here, there are 10 golden comparisons in a single test, if I ran the tests normally, only one of the 10 images shows up in the triage page. So instead of putting all the comparisons in a single test, I split them up temporarily into multiple tests, so that all the new images are uploaded in a single run, and I can triage all of them at once, instead of rerunning the test 10 times :)

Reply:
Resolve to this since it is commenting on a temporary change.